src/lib/components/PostBox.svelte

```diff
@@ -0,0 +1,48 @@
+<script lang="ts">
+  import { makePostService } from '$lib/services/post';
+  import { userStore } from '$lib/stores/user';
+
+  import Avatar from './Avatar.svelte';
+  import Card from './Card.svelte';
+  import Input from './Input.svelte';
+  import Button from './Button.svelte';
+
+  import { Scope } from '$lib/graphql/schema';
+
+  let content = '';
+
+  let customClassName: string = null;
+  export { customClassName as class };
+
+  async function createPost() {
+    if (content === '') {
+      return;
+    }
+    const postService = makePostService();
+    await postService.createPost({
+      content,
+      scope: Scope.Public
+    });
+  }
```

Comment:
Instead of using a custom approach here, I think we could use `manzana` for this as it is the common approach application wide. Manzana takes care of form common patterns and Yup of validation. You can find examples in the `Authenticate` component

Reply:
ok!